### DISCUSSION TITLE
Weird issues with my yabai setup - windows are jumping around or shaking when resizing/creating

### DISCUSSION THREAD

**Discussion by Lemon-2333** [asmvik/yabai#2381] [Q&A]
The issue is just like in this video - the windows are randomly jumping around all over the place when I try to resize or create new ones. I haven't installed any other window management software besides yabai, so I'm not sure what could be causing this strange behavior.

https://github.com/user-attachments/assets/daa9460c-2dc8-4b26-af79-5795ffa90813

**Comment by Bellavene** [comment]
Something is off in your config files

**Reply by Serubin** [reply]
Removing `yabai -m config layout                       bsp` resolved the issue for me

**Reply by Bellavene** [reply]
Yeah, because something is interfering, you literally turned off one instance of yabai, it looks so. Check for another instance or for another tiling manager in memory.

**Reply by Serubin** [reply]
@Bellavene What do you mean one instance of yabai was turned off? The command I removed from the config does not spin up a yabai instance. It's a config setter.

There is an issue with the bsp layout when explicitly defined as a part of the config.

**Reply by Bellavene** [reply]
I mean that something is racing to take control of the windows, that's what is seen in your video. Either another instance of a window manager, or a broken signal in .yabairc
`yabai -m config layout` should be `bsp` if you need tiling.

**Reply by Serubin** [reply]
Sure – but to clarify, there is only one instance of Yabai running on my machine.

Yabai has a built-in locking mechanism that prevents multiple processes from spawning, so it is unlikely that I (or others seeing this bug) have duplicate instances active. This points to a race condition in the software rather than user error.

Removing the explicit layout config resolved the issue on two of my machines. Since Yabai defaults to BSP anyway, removing that line is a safe fix that seems to bypass the race condition triggered by the config option.

**Comment by hmepas** [comment]
Same issue. Changing settings didn't helped. I had layout stack, but removing it didn't helped.  :(
Yabai really brocken on Tahoe. And Tahoe itself also doesnt feel good tbh..

**Reply by hmepas** [reply]
Also tried to complete remove yabairc. BSP still broken, windows jumping like crazy.

**Comment by aerickson** [comment]
Just encountered this behavior. In my case it was due to having two instances of yabai.

**Reply by nepthar** [reply]
How did you resolve it?

**Reply by aerickson** [reply]
In my case it was due to old homebrew installs. I wrote a small cleanup script.

https://gist.github.com/aerickson/f2e32e852f629c5f33ba49649b8baeb5

**Reply by hmepas** [reply]
> How did you resolve it?

`pkill yabai `
then start it again.
